taxi/views.py

```diff
@@ -87,3 +88,58 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+    template_name = "taxi/driver_form.html"
+    form_class = DriverCreateForm
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin,
+                              UserPassesTestMixin,
+                              generic.UpdateView):
+    queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/license_form.html"
+
+    def get_success_url(self):
+        return reverse_lazy("taxi:driver-detail",
+                            kwargs={"pk": get_user(self.request).pk})
+
+    def test_func(self):
+        driver = self.get_object()
+        return self.request.user.id == driver.id
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["user"] = self.request.user
+        return context
+
+    def handle_no_permission(self):
+        return render(self.request, "taxi/license_form.html", {
+            "error_message": "You are not allowed to "
+                             "update another driver's "
+                             "license number!",
+        })
```

Comment:
The `handle_no_permission` method in `UserPassesTestMixin` should return an HTTP response with a status code indicating permission denial, such as `HttpResponseForbidden`. Consider using `HttpResponseForbidden` instead of rendering a template with an error message.